internal/broker/broker.go

```diff
@@ -70,12 +70,15 @@ type session struct {
 
 	oidcServer   *oidc.Provider
 	oauth2Config oauth2.Config
-	authInfo     map[string]any
```

Comment:
The idea behind this was to keep the session as dynamic as possible without having to define a lot of other fields in the type. That was, of course, before knowing that we would only have Device Authentication available as a provider authentication mode.

Reply:
we can still add more fields to the session if at some point we support other authentication modes than device auth